src/nix/main.cc

```diff
@@ -126,7 +126,7 @@ struct NixArgs : virtual MultiCommand, virtual MixCommonArgs
         {"ls-store", {"store", "ls"}},
         {"make-content-addressable", {"store", "make-content-addressed"}},
         {"optimise-store", {"store", "optimise"}},
-        {"ping-store", {"store", "ping"}},
+        {"info-store", {"store", "info"}},
```

Comment:
Thanks.  So, According to the comment, the ping-store need to stay the same but, the vector alias of it `{"store", "info"}` can be as it is.  please let me know if this is the expectation

Reply:
I do not off-hand know of a mechanism for aliasing within subcommands, but if we look how this is implemented with overriding `rewriteArgs`, perhaps that can be done within the `Store` command.

Reply:
I am not sure what you are saying? I am saying you can modify `src/nix/store.cc` to that `nix store ping` is an alias for `nix store info` in addition.

Reply:
Please let me know if there are any documentations available for this aliasing.

Reply:
There is no documentation for this exact task. You would need to figure out what `main.cc` is doing and recreate it in `store.cc`. I am sorry.

It would be good to add API docs to `rewriteArgs` where it is declared (`args.hh`). Perhaps once you've figured out what is going on your can do that, so the next person is less stuck?

Reply:
Sure.  I will try to implement it in store.cc.  Thanks.